packages/kmod/kmod.spec

```
@@ -44,6 +65,11 @@ done
 install -d %{buildroot}%{_cross_sbindir}
 ln -s ../bin/kmod %{buildroot}%{_cross_sbindir}/modprobe
 
+pushd static-build
+install -d %{buildroot}%{_cross_libexecdir}
+cp tools/kmod %{buildroot}%{_cross_libexecdir}/kmod
```

We're not like, super close to reproducible builds, but this is a baby step to avoid gratuitous timestamp changes:

```suggestion
install -p -m 0755 tools/kmod %{buildroot}%{_cross_libexecdir}/kmod
```

---

Makes sense. Had to play with some flags but `install -p -m 0755 -D tools/kmod -t %{buildroot}%{_cross_libexecdir}/kmod/kmod` seems to do everything we want